helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -487,9 +484,19 @@ or modify your existing configuration to migrate to a new Auth0 Tenant.
 ### From FiftyOne Teams Version 1.6.0
 
 1. Ensure all FiftyOne SDK users either
-    - set `FIFTYONE_DATABASE_ADMIN=false`
-    - `unset FIFTYONE_DATABASE_ADMIN`
-        - This should generally be your default
+    - Set the `FIFTYONE_DATABASE_ADMIN` to `false`
+
+      ```shell
+      FIFTYONE_DATABASE_ADMIN=false
+      ```
+
+    - Unset the environment variable `FIFTYONE_DATABASE_ADMIN`
+      (this should generally be your default)
+
+        ```shell
+        `unset FIFTYONE_DATABASE_ADMIN`
```

Comment:
I don't think you want single quotes in a code block?

Reply:
Yes I do : )

Reply:
Good catch. they aren't single quotes, but backticks. Either way, they don't belong.